README.md

```diff
@@ -30,16 +30,21 @@
 </div>
 
 <!-- ABOUT THE PROJECT -->
+
```

Comment:
@hamza-akram: Please remove the extra new lines added in the whole file

Reply:
Done!

Reply:
@hamza-akram: Still this new line remains

Reply:
Done